string dimension
float64 value
lvr_ros/Mesh mesh
---
float64[] segments
geometry_msgs/Point[] points
